portfolio/src/main/webapp/requests/queries/index.js

```diff
@@ -0,0 +1,24 @@
+const get_comments = async () => {
+    const payload = await fetch('/comments');
+    const comments = await payload.json();
+
+    const comments_section = document.getElementById("comments");
+    comments.forEach(comment_payload => {
```

Comment:
nit: The Google Javascript Style Guide [suggests](https://google.github.io/styleguide/jsguide.html#naming-parameter-names) that parameter names are lowerCamelCased, i.e. `commentPayload`